### ISSUE TITLE
Is npm auditing packages that are published

### ISSUE THREAD

**Issue by mrveera** [npm/registry-issue-archive#296]
Hey team,
I want to know whether npm auditing packages or not.
if not, I have an idea to audit  existing packages and the new packages as well
I am ready to work on this issue but I need help from the team.

**Comment by amaramth** [comment]
how could it? there are very many packages

**Comment by mrveera** [comment]
Yeah. There are very many packages but we can still audit by using the following method I think 
For every publish of npm package publish.We can audit it for malicious code before publishing to the registry.
Every time someone update package it will be audited automatically.
For old packages, we can do the similar thing but it will be little complex
